### ISSUE TITLE
Fellglow Keep Unleveled Enemies

### ISSUE THREAD

**Issue by ThrasherGX** [jdsmith2816/eldersouls#89]
**Describe the bug**
Fellglow Keep (One of the locations used by the Mage Guild questline) is guarded by very powerful enemies, including some at/over level 40.  As this is a main questline location, these enemies should probably be leveled, since major quests should not have such harsh level restrictions.

**Expected behavior**
Major quest locations should not be blocked by unleveled enemies.  Any enemies around these locations should be appropriately leveled to ensure that players can reasonably complete these questlines.

**Modlist version**
1.1.4
**Did you add any additional mods?**
No

**Comment by jdsmith2816** [comment]
Agreed.  I really need to do a pass on these unleveled enemies soon.  It's gonna be a while before enxt update though.. I'm in the middle of reworking the housing merge and my time is limited lately.
